ui/app/components/keyboard-shortcuts-modal.js

```diff
@@ -28,7 +27,7 @@ export default class KeyboardShortcutsModalComponent extends Component {
    * hints: filter keyCommands to those that have an element property,
    * and then compute a position on screen to place the hint.
    */
-  @computed('keyboard.keyCommands.length', 'keyboard.displayHints')
+  @computed('keyboard.{keyCommands.length,displayHints}')
```

Comment:
`question`: Why not just declare `length` and `displayHints` as tracked properties here? Are we relying on `computed` properties for a cache?

Reply:
Could you expand on this a bit? What is the benefit to having these as tracked properties of the modal vs global state in the service?

Reply:
This is a Glimmer component and the long-term plan is moving to a pull-based reactivity model (tracked properties) and fully away ([deprecating](https://github.com/emberjs/rfcs/issues/536)) from a push-based reactivity model (computeds, observables, two-way bindings).